statemachine/event.py

```diff
@@ -41,17 +39,18 @@ def _process(self, trigger_data: TriggerData):
                 event_data.executed = True
                 break
         else:
-            raise TransitionNotAllowed(trigger_data.event, state)
+            if not trigger_data.machine._queued:
+                raise TransitionNotAllowed(trigger_data.event, state)
```

Comment:
author's note: This decision assumes that if we're in a queued mode, we don't expect that receiving an unknown or invalid event raises an exception. We do care that the state machine keeps in a valid state.

Should this deserve its own flag?